index.html

```diff
@@ -211,10 +210,10 @@ <h2>Survey</h2>
                 <label>Do you like radio buttons?</label>
                 <label><input type="radio" name="decision" value="yes">Yes</label>
                 <label><input type="radio" name="decision" value="no">No</label>
-                <label><input type="radio" name="decision" value="?" selected>None of your business</label>
+                <label><input type="radio" name="decision" value="?" checked>None of your business</label>
                 <label for="life_meaning">What is the meaning of Life?</label>
-                <input type="text" name="lifesplaination" id="life_meaning" placeholder="Enter the correct answer here">
-                <input type="button" value="SEND" class="send">
+                <textarea rows="10" name="life_meaning" id="life_meaning" placeholder="Enter the correct answer here"></textarea>
+                <input type="submit" value="SEND" class="send">
```

Comment:
Incorrect element
Missing rows attribute
Incorrect name
Incorrect type for submit button